evalml/tests/component_tests/test_simple_imputer.py

```diff
@@ -353,86 +362,93 @@ def test_simple_imputer_with_none():
     assert_frame_equal(expected, transformed, check_dtype=False)
 
 
-def test_simple_imputer_supports_natural_language_constant():
+@pytest.mark.parametrize("na_type", ["python_none", "numpy_nan", "pandas_na"])
+@pytest.mark.parametrize("data_type", ["Categorical", "NaturalLanguage"])
+def test_simple_imputer_supports_natural_language_and_categorical_constant(
```

Comment:
Since these tests failed, I just refactored them to be more descriptive per the parameters in case they should fail again.